usim/_primitives/activity.py

```diff
@@ -51,16 +51,37 @@ def __transcript__(self) -> ActivityCancelled:
 
 
 class ActivityExit(BaseException):
-    ...
+    """A :py:class:`~.Activity` forcefully exited"""
 
 
-class Activity(Condition, Generic[RT]):
+class Activity(Awaitable[RT]):
     """
-    Active coroutine that allows others to listen for its completion
+    Concurrently running activity that allows multiple activities to await its completion
```

Comment:
I think the word *activities* is misleading here. Maybe better *multiple objects including activities*?